src/components/sidebar/right-sidebar/RightSidebar.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useVideos } from '../../../context/videos-context/VideoContext';
+import { VideoCard } from '../../card/video-card/VideoCard';
+
+const RightSidebar = () => {
+  const { videos } = useVideos();
+
+  return (
+    <div>
+      {videos.slice(0, 6).map((video) => (
+        <VideoCard video={video} />
```

Comment:
You can pass video prop as `{...video}` and while accepting props you can write it as 
```
const VideoCard = ({
    videoThumbnail,
    title,
    uploadDate,
    uploadDate = 1648333768273,
    views,
    channelName,
    channelImage,
    youtubeId,
  }) => {}
```
You won't need to pass key in that manner and the code would also look clean.